src/Core/View.js

```diff
@@ -79,6 +82,44 @@ function View(crs, viewerDiv, options = {}) {
     if (__DEBUG__) {
         this.isDebugMode = true;
     }
+
+    if (options.loadingScreen !== false) {
+        let loadingScreenContainer;
+        if (!options.loadingScreen || options.loadingScreen === 'itowns') {
+            const node = document.createElement('style');
+            node.innerHTML = LoadingScreenCSS;
+            document.body.appendChild(node);
+            // loading screen
+            viewerDiv.insertAdjacentHTML('beforeend', LoadingScreenHTML);
+            loadingScreenContainer = document.getElementById('itowns-loader');
+        } else if (typeof (options.loadingScreen) === 'string') {
+            loadingScreenContainer = document.getElementById(options.loadingScreen);
+        } else if (options.loadingScreen instanceof (HTMLElement)) {
+            loadingScreenContainer = options.loadingScreen;
+        } else {
+            throw new Error('Invalid value for options.loadingScreen. Expected: false, string or HTMLElement');
+        }
+
+        const view = this;
+        // auto-hide in 3 sec or if view is loaded
+        const hideLoader = () => {
+            if (!loadingScreenContainer) {
+                return;
+            }
+
+            loadingScreenContainer.style.opacity = 0;
+            loadingScreenContainer.style.pointerEvents = 'none';
+            loadingScreenContainer.style.transition = 'opacity 0.5s cubic-bezier(0.55, 0.085, 0.68, 0.53)';
+
+            setTimeout(() => {
```

Comment:
Should this use a `transitionend` event instead? (except, well, if we leave it to the CSS, as suggested above, then there might not even be a transition)
```js
loadingScreenContainer.addEventListener('transitionend', e => {
  viewerDiv.removeChild(e.target);
});
// using e.target allows us to eager 'null' the variable to prevent re-running this code
loadingScreenContainer = null;
```